### DISCUSSION TITLE
Any settings required for recipe search?

### DISCUSSION THREAD

**Discussion by wlsnmrk** [TomBursch/kitchenowl#309] [Q&A]
I've just set up a self-hosted instance and so far pretty much everything seems to be working, _except_ recipe search - even when I search for the first word in a recipe's title, I get "No recipes found :(" in the results. Wondering if I missed a setting somewhere?

**Comment by TomBursch** [comment]
Event when you search for the first letter of a recipe? This looks like a bug, if you have the time, please create an issue with some logs (or send them here).

**Comment by wlsnmrk** [comment]
Yep, even first letter of a recipe name gives me "No recipes found." Would be happy to create an issue, but where would I find the logs? (Sorry for asking, this feels like it should be obvious -- but I can't find any documentation on it, or anything in the interface or on the server.)

**Reply by TomBursch** [reply]
If you use docker compose, you can run `docker compose logs` which should print all logs.

**Reply by wlsnmrk** [reply]
Thanks! Here's a short log of a couple searches, from a fresh `compose up` of the server. It doesn't immediately suggest anything to me, so I wanted to see if it seemed helpful before I open an issue. ("Apple" and "chicken" are the first words in the titles of a few recipes in the database.)
[search-bug-redacted.log](https://github.com/TomBursch/kitchenowl/files/13602215/search-bug-redacted.log)

**Reply by TomBursch** [reply]
Hmm, the logs look normal, I guess you're using Safari? Does the same problem occur in other browsers or the mobile iOS/android app?

**Reply by wlsnmrk** [reply]
This was actually captured using Vivaldi on Windows as the client, which is Chromium under the hood (edit: which uses an engine derived from Apple WebKit, apparently). But yes, the same happens in Firefox and in mobile Safari.

**Reply by TomBursch** [reply]
You're right, this happens in any browser and any setup. It will be fixed in the next release.
